controller/linodecluster_controller.go

```diff
@@ -325,3 +366,75 @@ func (r *LinodeClusterReconciler) SetupWithManager(mgr ctrl.Manager, options crc
 func (r *LinodeClusterReconciler) TracedClient() client.Client {
 	return wrappedruntimeclient.NewRuntimeClientWithTracing(r.Client, wrappedruntimereconciler.DefaultDecorator())
 }
+
+func (r *LinodeClusterReconciler) addMachineToLB(ctx context.Context, clusterScope *scope.ClusterScope) error {
```

Comment:
#470 tries to move these helper functions into the appropriate helpers.go file or the cloud/scope/ directory if possible. Can we do that for these? 🙏

Reply:
Done